src/sentry/apidocs/decorators.py

```diff
@@ -0,0 +1,13 @@
+from .preprocessor import PUBLIC_ENDPOINTS
+
+
+def declare_public(methods):
```

Comment:
drf-spectacular by default wants to document all endpoints, so this is my strategy of making endpoints public. (decorator registry combined with preprocessing)